CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ This changelog follows [keepachangelog](https://keepachangelog.com/en/1.0.0/) fo
 This project versioning is _similar_ to [Semantic Versioning](https://semver.org), and is documented in [SEP 14](https://github.com/saltstack/salt-enhancement-proposals/pull/20/files).
 Versions are `MAJOR.PATCH`.
 
-### 3000.1
+## 3000.1
```

Comment:
We already released 3000.1 so this would make it into Sodium (30001)

Reply:
@Ch3LL Done, I had to prepare the section for `3001` as well.  I took the liberty to move my other entry to `3001` (#56237), since that was merged after the release of `3000.1` as well.